src/platform/site-wide/feature-toggles/reducers/index.js

```diff
@@ -14,12 +14,16 @@ export const FeatureToggleReducer = (state = INITIAL_STATE, action) => {
         loading: true,
       };
     case FETCH_TOGGLE_VALUES_SUCCEEDED:
+      window.VetsGov = window.VetsGov || {};
```

Comment:
Does this mean, we will have 2 ways of getting feature flags? the redux store way and through the window object?

Reply:
Yes, at least initially. Eventually we could make the window way the "official" way.

Reply:
Decided to go with grabbing the redux store from `startApp` instead.